src/handbook/company/security/third-party-risk-management.md

```diff
@@ -0,0 +1,59 @@
+---
+navTitle: Third-Party Risk Management Policy
+---
+
+# Access Control Policy
+
+| Policy owner   | Effective date |
+| -------------- | -------------- |
+| @ZJvandeWeg    | 2023-06-01     |
+
+## Purpose
+
+To ensure protection of the organization's data and assets that are shared with,
+accessible to, or managed by suppliers, including external parties or third-party
+organizations such as service providers, vendors, and customers, and to maintain
+an agreed level of information security and service delivery in line with supplier
+agreements.
+
+## Scope 
+
+All data and information systems owned or used by FlowForge that are business
+critical and/or process, store, or transmit Confidential data.
+
+
+## Policy
+
+A list of approved vendors/partners must be maintained and reviewed annually. This
+list will be linked from the [Vendor section of the handbook](/handbook/operations/vendors/).
```

Comment:
Let's link in this PR

Reply:
Done - spreadsheet created with an initial list based on the Saas Spend spreadsheet and added link to the vendor section.